software/messaging/src/main/resources/catalog.bom

```diff
@@ -25,6 +25,56 @@ brooklyn.catalog:
         type: org.apache.brooklyn.entity.messaging.activemq.ActiveMQBroker
         name: ActiveMQ Broker
         description: ActiveMQ is an open source message broker which fully implements the Java Message Service 1.1 (JMS)
+    - id: org.apache.brooklyn.entity.messaging.activemq-artemis
+      version: "1.0.0-SNAPSHOT" # BROOKLYN_VERSION
+      item:
+        type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
```

Comment:
Im very new to brooklyn, whats the difference between this and a concrete type? It be good to have a concrete type? like the others?